packages/vite/src/node/plugins/workerImportMetaUrl.ts

```diff
@@ -180,7 +180,8 @@ export function workerImportMetaUrlPlugin(config: ResolvedConfig): Plugin {
           s.update(
             urlIndex,
             urlIndex + exp.length,
-            `new URL(${JSON.stringify(builtUrl)}, self.location)`,
+            // add `'' +` to skip vite:asset-import-meta-url plugin
+            `new URL('' + ${JSON.stringify(builtUrl)}, import.meta.url)`,
```

Comment:
`/foo` gets replaced with `/@fs/foo` by vite:asset-import-meta-url plugin. Maybe we should add `/* @vite-ignore */` for `new URL` as well?

Reply:
Using `/* @vite-ignore */` works for me 👍

Reply:
I'll make a separate PR for that.

Reply:
Isn't that changing this code to use `/* @vite-ignore */` instead, or is there more to it?

Reply:
`new URL(foo, import.meta.url)` doesn't support `/* @vite-ignore */` now. So adding support for that is needed.

Reply:
Ah okay I thought it's already supported. Good to merge then.